Makefile

```diff
@@ -1,12 +1,11 @@
 PROJECT_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 include $(abspath $(PROJECT_DIR)/build/automation/init.mk)
 
+DOS_DB_FILE=$(shell aws s3 ls s3://nhsd-texasplatform-service-dos-lk8s-nonprod | sort | grep "dos-pg-dump-.*-clean-PU.sql.gz" | tail -n 1 | awk '{ print $$4 }')
```

Comment:
This expression is evaluated every time we run Makefile functionality but only used in the `download-sql-dump`. Please, can we move it to that target?